README.md

```diff
@@ -106,10 +132,10 @@ Write this grammar using EBNF with common extensions
 
   (Hint: my answer has seven states including the start and end states)
 
-
+   
 ## A2.3
 
-«replace this with your answer»
+https://github.com/fidelianawar/cs3342_09/blob/master/Q2.3%20FSM.pdf
```

Comment:
First of all, you have the best handwriting of any diagram so far!

The diagram is _almost_ right. The only problem is that empty string between S1 and S2. You can't have a transition without an event. But all you need to do is combine S1 and S2:

![image](https://user-images.githubusercontent.com/10648/78718994-d46ed300-78e8-11ea-846e-7e63a8be291b.png)


# -2